src/Functions/visitLink.spec.js

```diff
@@ -0,0 +1,25 @@
+/// <reference types="jest" />
+
+import { visit } from '../testing';
+import Link from '../Models/Link';
+import { dropTable } from '../helpers';
+
+beforeEach(() => dropTable(Link));
+
+describe('visitLink', () => {
+  test('It should redirect', async () => {
+    await Link.create({ key: 'xyz828f', url: 'http://vote.dosomething.org' });
+
+    const response = await visit(`/xyz828f`);
+
+    expect(response.status).toBe(302);
+    expect(response.headers.location).toBe('http://vote.dosomething.org');
+  });
+
+  test('It should handle missing links', async () => {
+    const response = await visit(`/xyz123/info`);
```

Comment:
Why would we test for the `/info` here and not just a shortlink that doesn't exist? 

I might just be misreading the intention of the test...  just confused since the `/info` isn't a thing at this stage in the development.

Reply:
Oh `/info` comes in #61 haha. Still I kind of found the test misleading since I would expect it to just test the existence of the shortlink and that if it's missing it redirects to 404, not that the `/info` route is potentially broken but the shortlink exists.

Reply:
Oh no that's totally a mistake! Should be checking for the `/{link}` 404ing.

Reply:
Fixed in e182d5b.